pkg/service/appvendor/handler.go

```diff
@@ -0,0 +1,89 @@
+package appvendor
+
+import (
+	"context"
+
+	"openpitrix.io/openpitrix/pkg/gerr"
+	"openpitrix.io/openpitrix/pkg/logger"
+	"openpitrix.io/openpitrix/pkg/models"
+	"openpitrix.io/openpitrix/pkg/pb"
+	"openpitrix.io/openpitrix/pkg/util/pbutil"
+)
+
+type Handler struct {
+}
+
+func (h *Handler) DescribeVendorVerifyInfos(ctx context.Context, req *pb.DescribeVendorVerifyInfosRequest) (*pb.DescribeVendorVerifyInfosResponse, error) {
+	resource := Resource{}
+	vendors, count, err := resource.DescribeVendorVerifyInfos(ctx, req)
+
+	if err != nil {
+		return nil, gerr.NewWithDetail(ctx, gerr.Internal, err, gerr.ErrorDescribeResourcesFailed)
+	}
+	var vendor models.Vendor //need use a vendor object to call function
+	vendorPbSet, err := vendor.ParseVendorSet2pbSet(ctx, vendors)
+	if err != nil {
+		return nil, err
+	}
+	res := &pb.DescribeVendorVerifyInfosResponse{
+		VendorVerifyInfoSet: vendorPbSet,
+		TotalCount:          count,
+	}
+	return res, nil
+}
+
+func (h *Handler) GetVendorVerifyInfo(ctx context.Context, req *pb.GetVendorVerifyInfoRequest) (*pb.VendorVerifyInfo, error) {
+	userID := req.GetUserId()
+	resource := Resource{}
+	vendor, err := resource.GetVendorVerifyInfo(ctx, userID)
```

Comment:
if `VendorVerifyInfo` not exists, create new and return, instead of return error directly.